### PR TITLE
Gum vandalism

### PR DESCRIPTION
**Pull request by help-maint**
dreamseeker_03_05_2021_15_36_36_45
dreamseeker_03_05_2021_18_53_53_30
Did you hear about the astronaut who stepped in gum?
He got stuck in orbit!

:cl:
 * rscadd: You can now stick chewed gum under chairs, and in the coin slot of vending machines. Stuck gum can be removed with screwdrivers or acid.

### CODE REVIEW

**Comment by Zth--** [comment]
Holy fuck I was gonna work on this

**Comment by BomberBro** [comment]
can the gum sprite on the vending machine be a bit smaller?

**Comment by help-maint** [comment]
It's the actual sprite of the gum itself

File: code/game/objects/items/gum.dm [hunk 1]
```diff
@@ -143,15 +164,20 @@
 	wrapped = FALSE
 	update_icon()
 
+/obj/item/gum/attackby(var/obj/item/W, var/mob/user)
+	if(locked_to)
+		var/datum/locking_category/category = locked_to.get_lock_cat_for(src)
+		if(istype(category, /datum/locking_category/gum_stuck) && is_type_in_list(W, list(/obj/item/weapon/chisel, /obj/item/tool/screwdriver)))
```
Comment (Zth--) [inline code/game/objects/items/gum.dm]: I'd like to see this with a do_after delay so it takes some time to remove the gum (making it extra annoying)

**Comment by BomberBro** [comment]
yeah but it's xboxhuge, looks like somebody put a year's worth of chewing on the machine

**Comment by help-maint** [comment]
Do you know of any items that resize themselves, if so I can copy that code, otherwise 2much work

**Comment by BomberBro** [comment]
however the hell dehydrated monkeys/humans work to shrink themselves with the correct sprites, otherwise I got nothing

**Comment by Zth--** [comment]
does lock_atom() fail if you try to put chewed gum into an already vandalized object? What happens when you try to put more chewed gum into it?

**Comment by help-maint** [comment]
You stick multiple gums. You monster.

**Comment by gurfan** [comment]
guillotines resize people when placed into them, I'd check how those things do it

**Comment by Zth--** [comment]
> You stick multiple gums. You monster.

Then it'd be a good idea to randomize the gum location a bit so it looks like there's multiple gum stacked instead of only one. At least for vending machines. It won't work for chairs/stools

**Comment by KingofHONK** [comment]
I am already so thankful for this, but I beg you, please add sticking gum to people's hair. I need it for reasons

**Comment by Blithering** [comment]
Mob cubes resize whatever mob they were when they get deformed, and any item in a display case resizes them. Might be places to look.

**Comment by beesil** [comment]
Gun sprites are also scaled down to go into rechargers.

**Comment by help-maint** [comment]
dreamseeker_03_05_2021_18_53_53_30
Done adding stuff to this branch, pending code review, new stuff can go on a followup PR

**Comment by Eneocho** [comment]
Is this the resurrection of the explosive gum? Take the explosive gum, bite, place under target's chair, target explodes.

**Comment by help-maint** [comment]
Explosive gum can already be "attached" to anything, chairs included, it's just invisible

**Comment by Eneocho** [comment]
> 
> 
> Explosive gum can already be "attached" to anything, chairs included, it's just invisible

I meant that it'd be a player-side revival of the idea, since you now have a plausible reason to stick gum somewhere, you were just eating some.

**Pull request merged**